src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { PageNotFound } from './components/PageNotFound/PageNotFound';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route path=":slug" element={<PeoplePage />} />
```

Comment:
You may have made a mistake here, as you're rendering the `PeoplePage` component for both the '/people' and '/people/:slug' routes. This could lead to unexpected behavior. If you have a separate component for individual people, you should render that here instead.